draft-ietf-quic-applicability.md

```diff
@@ -446,6 +446,17 @@ leakage of information about the data that is sent. A QUIC implementation can
 expose an interface that allows an application layer to specify how to apply
 padding.
 
+# ACKing on constraint links
+
+On network paths where there is significant asymmetry,
+the transmission of QUIC ACKs can reduce the efficient use of network
+capacity or transmission opportunities. This effect occurs
+when the return capacity is significantly more constrained than the
```

Comment:
I don't believe return and forward are used elsewhere, but this sentence also seems a bit redundant to the previous sentence which discusses asymmetric paths.

Reply:
Maybe instead:

> The cost of sending acknowledgments - in processing cost or link utilization - could be a significant proportion of available resources if these resources are constrained.  Reducing the rate at which acknowledgments are generated can preserve these resources and improve overall performance.

Reply:
This text (or similar) seems OK to me.

Is "overall" here meaning QUIC user performance  (or overall link-layer?) I think impact on performance for the QUIC user is the more significant additional take-away.

Reply:
Overall was meant to be all-inclusive, which definitely includes application-relevant metrics like time to first byte, total transfer time, transfer rate, ....  It's clear that there are many ways in which this might help.

Reply:
We agree. Although my question is, it is obvious to a QUIC user that there user performance can be impacted - for several decades they could have used TCP, where other mitigations exist? So can we add "(including application-relevant metrics)" or something?